vault.go

```diff
@@ -171,7 +171,128 @@ func doMinioRole(vc *vault.Client, authPath, name string) error {
 	return nil
 }
 
-func doVaultConfiguration(vc *vault.Client, profileName string, minioInstances []string, kubernetesAuthPath string) error {
+func contains(s []string, e string) bool {
```

Comment:
There is already a helper for this: https://github.com/StatCan/kubeflow-controller/blob/master/utils.go#L4

Reply:
Ouuuu! Thanks!